resources/views/home.blade.php

```diff
@@ -4,30 +4,25 @@
 
 @section('content')
 
-    <img src="images\boot.PNG" class="rounded mx-auto d-block w-25" >
+    <img src="images\boot.PNG" class="rounded mx-auto d-block w-25 mt-3 mb-3">
```

Comment:
Instead of using both mt and mb you can use my, this will add both margin on the top and bottom